core/src/test/java/kafka/test/annotation/ClusterTestDefaults.java

```diff
@@ -35,7 +35,7 @@
 @Target({TYPE})
 @Retention(RUNTIME)
 public @interface ClusterTestDefaults {
-    Type clusterType() default Type.ZK;
+    Type[] clusterTypes() default {Type.ZK, Type.KRAFT, Type.CO_KRAFT};
```

Comment:
Maybe it should be renamed to `types` instead of `clusterTypes`. We do have a `ClusterType` in testing :)